distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -31,6 +31,9 @@ javax.net.ssl.trustStorePassword=changeit
 javax.net.ssl.keyStoreType=jks
 javax.net.ssl.trustStoreType=jks
 
+#force java to run in headless mode for when the server doesn't have a display device
+java.awt.headless=true
```

Comment:
What if the server does have a display device? Does this impact a standard distribution and its ability to interact with the display? I assume not, as we're mediating everything through the browser?